### PR TITLE
chore: fix: CRW-4070, CRW-4504 copy...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: fix: CRW-4070, CRW-4504 copy org.eclipse.che_checlusters.yaml to devspaces.crd.yaml so that new field gatewayContainer is present (along with other updates)

Change-Id: I363c6c189fdc5ed5b30c5c08404852db76ff1e68
Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: devspaces-operator-bundle/build/scripts/sync-che-olm.sh [hunk 1]
```diff
@@ -340,4 +340,8 @@ for CSVFILE in ${TARGETDIR}/manifests/devspaces.csv.yaml; do
 	done
 done
 
+# CRW-4070, CRW-4504 make sure upstream org.eclipse.che_checlusters.yaml content is copied downstream
+# TODO should this be config/crd/bases/org.eclipse.che_checlusters.yaml instead?
```
Comment (amisevsk) [inline devspaces-operator-bundle/build/scripts/sync-che-olm.sh:344]: Unless downstream is running the generators (olm, kustomize) then the correct resource to copy is the one below.

**Review by amisevsk — APPROVED** [review]
:+1:

**Comment by amisevsk** [comment]
This change is still (?) missing the related `sed`, unless this happens elsewhere in the script

```
sed -r -e "s#quay.io/devfile/universal-developer-image.+#registry.redhat.io/devspaces/udi-rhel8@${DIGEST}#g" -i \
	"${TARGETDIR}/bundle/${OLM_CHANNEL}/eclipse-che/manifests/org.eclipse.che_checlusters.yaml"
```

File: devspaces-operator-bundle/build/scripts/sync-che-olm.sh (hunk 1 above)
Comment (nickboldt) [inline devspaces-operator-bundle/build/scripts/sync-che-olm.sh:344]:
```suggestion
# NOTE: don't use config/crd/bases/org.eclipse.che_checlusters.yaml unless we're using generators (olm, kustomize) downstream
```

**Comment by nickboldt** [comment]
> This change is still (?) missing the related `sed`, unless this happens elsewhere in the script
> 
> ```shell
> sed -r -e "s#quay.io/devfile/universal-developer-image.+#registry.redhat.io/devspaces/udi-rhel8@${DIGEST}#g" -i \
> 	"${TARGETDIR}/bundle/${OLM_CHANNEL}/eclipse-che/manifests/org.eclipse.che_checlusters.yaml"
> ```

that was removed because `universal-developer-image` is now set elsewhere, rather than being hardcoded in the CRD.

**Pull request merged**